CIScripts/Test/TestConfigurationUtils.Tests.ps1

```diff
@@ -0,0 +1,82 @@
+. $PSScriptRoot\TestConfigurationUtils.ps1
```

Comment:
What do you think about enabling strict mode? It can be done by invoking Init.ps1 (see other tests).